packages/eslint-plugin/src/configs/core.js

```diff
@@ -0,0 +1,360 @@
+// @ts-check
```

Comment:
NOTE: this whole config is copy-paste of original `react.js` file , cleaned up by removing react family specific rules/plugins setup